test/client/setup-test-framework.js

```diff
@@ -41,6 +41,9 @@ jest.mock( 'wpcom-proxy-request', () => ( {
 	canAccessWpcomApis: jest.fn(),
 } ) );
 
+// Mock crypto.randomUUID with its Node.js implementation
+global.crypto.randomUUID = () => 'fake-uuid';
```

Comment:
That's not great because it'll always be the same and that might cause some weird behaviors. We might want to keep uniqueness in some tests and add further inline mocks that set unique UUIDs.